Core.lua

```diff
@@ -61,7 +61,17 @@ function BeardLib:Init()
 		end
 	end
 
-	self.Version = tonumber(self.AssetUpdates.version)
+	if self.Options:GetValue("GithubUpdates") then
+		local module = ModAssetsModule:new(self, {id = "simon-wh/PAYDAY-2-BeardLib", _meta = "AssetUpdates", important = true, provider = "github", branch = "master"})
+		self[module._name] = module
+		table.insert(self._modules, module)
+	else
+		local module = ModAssetsModule:new(self, {id = 14924, _meta = "AssetUpdates", important = true, provider = "modworkshop"})
+		self[module._name] = module
+		table.insert(self._modules, module)
+	end
+
+	self.Version = tonumber(self.AssetUpdates.Version) or tonumber(4.7) -- Set to the latest version if the version is not found, assuming it's a hash.
```

Comment:
I feel like it would be better to take the version from the xml rather than hardcoding it here